src/glib-backports.h

```diff
@@ -25,6 +25,52 @@
 
 #include <gio/gio.h>
 
+#if GLIB_CHECK_VERSION(2, 76, 0)
+
+#include <glib/gstdio.h>
```

Comment:
Doesn't this need to be included always? the else branch uses `g_close`

Reply:
Sure